bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/handler/VerisureBridgeHandler.java

```diff
@@ -125,12 +127,11 @@ public void initialize() {
                     VerisureSession session = this.session;
                     updateStatus(ThingStatus.UNKNOWN);
                     if (session != null) {
-                        if (!session.initialize(authstring, pinCode, config.username)) {
-                            logger.warn("Failed to initialize bridge, please check your credentials!");
+                        if (!session.initialize(authstring, pinCode, config.username, config.password)) {
+                            logger.warn(
+                                    "Failed to login to Verisure, please check your account settings! Is MFA activated!");
```

Comment:
No need fopr this log, the user has already the information with your call to `updateStatus`

Reply:
Removed it!